ros2_batch_job/osx_batch/__init__.py

```diff
@@ -38,6 +38,8 @@ def pre(self):
             os.environ['LANG'] = 'en_US.UTF-8'
         if 'ROS_DOMAIN_ID' not in os.environ:
             os.environ['ROS_DOMAIN_ID'] = '111'
+        if 'OPENSSL_ROOT_DIR' not in os.environ:
+            os.environ['OPENSSL_ROOT_DIR'] = '/usr/local/Cellar/openssl/1.0.2k'
```

Comment:
We should be able to move this to the bash_profile (of the henkins user) on each MacOS node and not have to put it in the ci script here